treeherder/perf/models.py

```diff
@@ -15,13 +15,17 @@
 class PerformanceFramework(models.Model):
 
     name = models.SlugField(max_length=255, unique=True)
+    enabled = models.BooleanField(default=False)
 
     class Meta:
         db_table = 'performance_framework'
 
     def __str__(self):
         return self.name
 
+    def __getitem__(self, enabled):
```

Comment:
Oh! You shouldn't access 'enabled' that way. Do it this way:

```py
if not framework.enabled:
```

Then you can take this out. :)